.github/workflows/e2e-test.yaml

```diff
@@ -70,22 +70,12 @@ jobs:
       - run: make docker-build
       - run: make docker-build-examples
 
-      - name: make local-cluster
-        run: make local-cluster
-        id: make_local_cluster
+      - run: make local-cluster
 
       - run: make deploy
       - run: make example-vms
 
-      - name: Run make e2e
-        run: |
-          # Increase inotify limits for dnsmasq to fix
-          #   "dnsmasq: failed to create inotify: No file descriptors available"
-          sudo sysctl fs.inotify.max_user_instances=8192
-          sudo sysctl fs.inotify.max_user_watches=524288
-          sudo sysctl -p
```

Comment:
This is no longer necessary?

Reply:
Yep, this has been added to runners itself: https://github.com/neondatabase/runner/pull/20